parse-jtl.py

```diff
@@ -158,6 +164,50 @@ def compare_csv(input_dict, baseline_dict, deviance_dict):
     return result, failures
 
 
+def generate_histograms(results):
+    rows = len(results.keys())
+    figure_num = 1
+    bin_size = 15  # Normalize to 15 buckets for each histogram
+
+    # figure height = 5 per row so that it is large enough
+    plt.figure(1, figsize=(20, 5 * rows))
+    for key, result in sorted(results.items()):
+        data_array = np.asarray(result['data'], dtype=np.integer)
+
+        # Plot the raw data
+        plt.subplot(rows, 2, figure_num)
+        plt.hist(data_array, bins=bin_size)
+        plt.title(key + " (sample size: {samples})\n avg = {avg}, median = {median}, "
+                        "std_dev={std_dev}".format(
+                            samples=len(data_array),
+                            avg=int(np.average(data_array)),
+                            median=int(np.median(data_array)),
+                            std_dev=int(np.std(data_array))
+                        ))
+        plt.grid(True)
+        figure_num += 1
+
+        # Remove outliers (over over 95 percentile) and plot that as well
+        percentile = int(np.percentile(data_array, 95))
+        data_array_minimized = data_array[data_array < percentile]
+
+        plt.subplot(rows, 2, figure_num)
+        plt.title(key + " up to 95 percentile (sample size: {samples})\n avg = {avg}, median = {median}, "
+                        "std_dev={std_dev}, percentile cutoff={percentile}".format(
+                            avg=int(np.average(data_array_minimized)),
+                            median=int(np.median(data_array_minimized)),
+                            std_dev=int(np.std(data_array_minimized)),
+                            percentile=percentile,
+                            samples=len(data_array_minimized)
+                        ))
+        plt.hist(data_array_minimized, bins=bin_size)
+        plt.grid(True)
+        figure_num += 1
+
+    plt.tight_layout()
+    plt.savefig("performance_graphs.pdf")
```

Comment:
Proposal: Do not use hard-coded file name "performance_graphs.pdf" but require usage of `--generate-histograms` with file name (remove line 60 and add some default value).